uqcsbot/dominos_coupons.py

```diff
@@ -44,50 +48,105 @@ async def dominoscoupons(
         number_of_coupons: int = 5,
         ignore_expiry: bool = True,
```

Comment:
Unrelated to the changes you have made, but it might be best to make this `False` by default. This should provide a better list of coupons,  at the risk of having less coupons to print out.

Reply:
Good call